optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/Joiners.java

```diff
@@ -139,9 +136,7 @@
     }
 
     /**
-     * Joins every A and B where a value of property on A is less than or equal to the value of the same property on B.
-     * These are exactly the pairs where {@code mapping.apply(A).compareTo(mapping.apply(B)) <= 0}.
-     * Delegates to {@link #lessThanOrEqual(Function, Function)} with both arguments using the same mapping.
+     * As defined by {@link #lessThanOrEqual(Function, Function)} with both arguments using the same mapping.
```

Comment:
Maybe I am a bit confused
In one line I see  
 never null, a joiner that filters the constraint stream to only include elements (A,B) where a property
But there is no B type in the definition 
 public static <A, Property_ extends Comparable<Property_>> BiJoiner<A, A> lessThanOrEqual(

Reply:
Hmm, good point. The A and B here are just other ways for saying "left" and "right". 

That said, since the Javadoc is now much more complete, we can probably just remove all the return descriptions, and keep there just the "never null" part.